src/routes/users.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { controllerDeleteUser,
+  controllerGetAllUsers,
+  controllerGetUserById,
+  controllerPostUser,
+  controllerUpdateUser } = require('../controllers/users');
```

Comment:
and here

Reply:
Here the same. It is a bad  practice to write `const {` and something after `{` on the same line.